local-cli/runAndroid/runAndroid.js

```diff
@@ -250,15 +250,15 @@ function startServerInNewWindow() {
   const procConfig = {cwd: packagerDir};
 
   if (process.platform === 'darwin') {
-    if (yargV.open) {
```

Comment:
Can we use `env` variable for consistency with other places where we already use it? 

Example: https://github.com/facebook/react-native/blob/dfaa03e1cb12b349152ca10f997ac198ab995d1e/local-cli/server/util/launchEditor.js#L84

`REACT_TERMINAL` or something might be a good idea for future scripts